plugins/search-react/src/components/SearchTracker/SearchTracker.tsx

```diff
@@ -16,10 +16,12 @@
 
 import React, { useEffect } from 'react';
 import { useAnalytics } from '@backstage/core-plugin-api';
-import { useSearch } from '@backstage/plugin-search-react';
+import { useSearch } from '../../context';
 
 /**
  * Capture search event on term change.
+ *
+ * @public
```

Comment:
Looks like this isn't actually exported from the package (which is probably good!).  Good to remove this `@public` for clarity though.